lib/client/ipam_test.go

```diff
@@ -103,6 +103,70 @@ type testArgsClaimAff struct {
 
 var _ = Describe("IPAM tests", func() {
 
+	Describe("IPAM AutoAssign from the default pool then delete the pool and assign again", func() {
```

Comment:
Maybe beef up comment to explain that we're testing you don't reuse affine block once pool is release.